code/style.css

```diff
@@ -1,9 +1,242 @@
-/* 
-If you can see a peach background, the css file is correctly linked. 
-Feel free to change or remove this styling! 
-We want you focus A LOT on the styling for this project. 
-*/
-
-body {
-  background-color: peachpuff;
-}
+body{
+  background-color:#FFF8CC;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size: 14px;
+  width:auto;
+  display: block;
+}
+
+h1{
+  color:#FFDE04;
+  font-size: 22px;
+  margin-top: 20px;
+  margin-bottom: 22px;
+}
+
+h2{
+  color: #CC6600;
+  font-size: 20px;
+  margin-top: 40px;
+  margin-bottom: 0;
+}
+
+h3{
+  color: #CC6600;
+  font-size: 16px;
+  margin-top: 20px;
+  margin-bottom: 0;
+}
+
+p{
+  width: 90%;
+  display: inline-block;
+}
+
+a{
+  color:#00ADDC;
+  font-style: normal;
+  text-underline-position: below;
+}
+
+div{
+  padding: 10px 0;
+}
+
+fieldset{
+  display: block;
+  align-items: baseline;
+  max-width: 50%;
+ ;
+}
+
+form{
+  margin-top: 10px;
+  max-width: 50%;
+
+}
+
+label{
+  padding-right: 10px;
+
+}
+
+mark{
+  color:black;
+  font-weight: bold;
+  background-color: inherit;
+}
+
+footer{
+  display: flex;
+  flex-direction: column;
+  align-items: left;
+  justify-content: left;
+  font-size: 8px;
+  margin: 30px 0;
+  padding: 10px 0px;
+}
+
+.Aufzählung{
+  display: flex;
+  align-content: space-between;  
+  padding: 0 4px;
+  background-color: #FFF8CC;
+  font-style: oblique;
+  font-variant: small-caps;
+}
+
+.contact-details{
+  display: flex;
+  padding: 5px 0;
+ 
+}
+
+
+.contact-field {
+  margin-right: 30px;
+  padding: 1px;
+}

```

Comment:
When you set `display: flex;` on the parent, it will cause the inputs to grow as much as its biggest sibling (the label text "Wohnort / evtl. Lieferort"). Consider adding a `max-height` to this `contact-field` for a better layout :)